### PR TITLE
Adding Slot Picker details

### PR DESCRIPTION
**Pull request by tomhazledine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,127 @@
...
+* `token` [required]: auth token for API connection. *Not required if the component is activated in demo mode.*
+* `demo`: boolean to activate demo-mode. Defaults to `false`. If `demo` is set to `true` the component will return mock data (and not make any API calls). 
+* `target` [required]: ID of mounting element (e.g. 'cronofy-slot-picker').
+* `api_domain` [optional]: Override the default Default value is `"http://api.cronofy.com"`.
+* `styles` [optional]: an object containing you custom style values See [Custom Styles](#custom-styles) for details of valid options.
+
+
+### Agenda options
+
+The Agenda component
```
Comment (tomhazledine) [inline README.md]: Not sure what happened here 😛 - should read:

"The Agenda component can be assigned a custom color palette for the calendars by passing in an array of HEX values to the `colors.calendars` option. For example:	`colors: { calendars: ['#0074d9','#ffdc00'] }`"

**Comment by tomhazledine** [comment]
My biggest concern with this doc at the moment is _coherence_:

* is it clear that this doc explains initialisation for *2* components?
* does it make sense to have a section for "global" instructions (common to all components) and then outline the component-specific parts afterwards? Or should there just be 1 section per component, with full instructions for each one (including a bit of duplication between components)?

**Comment by stephenbinns** [comment]
My 2p the duplication is probably the way to go.

I think you've explained both components, the examples are where most developers will look first and these look simple enough.

**Comment by tomhazledine** [comment]
I've switched around some bits:
* each component now has a self-contained install/init section.
* I've added an overview of token generation too.

**Pull request merged**